### ISSUE TITLE
[feature] E2EE support?

### ISSUE THREAD

**Issue by txchen** [bangle-io/bangle-io#427]
Hi, do you have plan to support E2EE? Currently we can save to github private repo, I think it would be even better if bangle can support encryption so users do not need to worry about leaking information to github.

**Comment by kepta** [comment]
Hey I am not fully sure I understand. Are you talking about syncing with another server which is fully encrypted ?

**Comment by txchen** [comment]
No, I mean encrypt the markdown content and save the encrypted content to github repo. 
User will need to provide a password/encryption_key to decrypt the content and edit it locally.
